scripts/generator.ts

```diff
@@ -34,38 +33,24 @@ const DISCOVERY_URL = argv['discovery-url'] ?
 const FRAGMENT_URL =
     'https://storage.googleapis.com/apisnippets-staging/public/';
 
-const API_TEMPLATE = './templates/api-endpoint.ts';
-const INDEX_TEMPLATE = './templates/index.ts';
-const BEAUTIFY_OPTIONS = {
```

Comment:
The previous code was reading in the contents of the template, and doing a render based on a string representation.  The new code just needs the file path to the template, and handles reading the file from disk for you :)  The beautify options went away because I got rid of beautify, and replaced it with clang-format.